erpnext/accounts/report/general_ledger/general_ledger.py

```diff
@@ -350,8 +354,12 @@ def update_value_in_dict(data, key, gle):
 			if filters.get("group_by") != _('Group by Voucher (Consolidated)'):
 				gle_map[gle.get(group_by)].entries.append(gle)
 			elif filters.get("group_by") == _('Group by Voucher (Consolidated)'):
-				key = (gle.get("voucher_type"), gle.get("voucher_no"),
-					gle.get("account"), gle.get("cost_center"))
+				keylist = [gle.get("voucher_type"), gle.get("voucher_no"), gle.get("account")]
+				if filters.get("include_dims"):
+					for dim in get_accounting_dimensions():
```

Comment:
Get accounting dimensions as list once and store in a variable. Use the same variable to loop here and generate `dimensions_fields` for queries

Reply:
OK, I have merged the two calls to get_accounting_dimensions to a single one up in get_results, and passed the resulting accounting_dimensions list down through the call chain to the two places it is used.